### ISSUE TITLE
V1.2.1 does not work at all

### ISSUE THREAD

**Issue by fappelman** [vknabel/vscode-swiftlint#13]
Hi,

I reinstalled vscode from scratch, removed `~/Library/Application Support/Code` and `~/.vscode`. Also removed stuff from caches and then installed just this extension. So this was the **only** extension installed. The version is `1.2.1`.

If I then open a `.swift` file nothing happens. (I know there are `lint` issues that should be flagged)
Also no errors etc.

I then downgraded to version `1.2.0`. 

I get one error on startup being:

```log
An unknown error occurred. spawn E2BIG

Source: SwiftLint (Extension)
```

One I clicked that error away everything works. So I get my expected lint warnings.

I then upgraded again to `1.2.1` and then it stops working again. No error and a silent `lint` output.

I also moved my `.swiftlint.yml` file outside my tree to see if that had any effect. It doesn't. 

Let me know if I need to provide more information.

**Comment by vknabel** [comment]
@fappelman Sorry for the late response, but thanks for the report. Had quite a lot to do recently. 

Can you insert the `includes` and `excludes` parts of your swiftlint config?
I try to have a deeper look this weekend.

**Comment by fappelman** [comment]
No problem with the response time. I have been using `1.2.0` in the mean time.

My `Force Exclude Paths` have the following values:

```log
tmp
build
.build
Pods
Carthage
.build-linux
.build-macos
.macos
.linux
````

There is no `includes` section in the config. Is that perhaps the problem?

I see `Config Search Paths`, `Enable`, `Force Exclude Paths`, `Only Enable On Swift PMProjects` and `Path`.

**Comment by vknabel** [comment]
Sorry for the delay and inconvenience. Quite a lot to do lately.
I hope [1.2.2](https://github.com/vknabel/vscode-swiftlint/releases/tag/1.2.2) fixes your issue! Can you confirm?

*How did this even work with 1.2.1? 😅*

**Comment by fappelman** [comment]
Just did a very small test. That seems to work. Will let you know if I run into issues. Thanks.

**Comment by vknabel** [comment]
Thanks, great to hear!
